src/main/java/testapp/adapter/memory/InMemoryUserRepository.java

```diff
@@ -26,9 +25,9 @@ public InMemoryUserRepository() {
   }
 
   @Override
-  public User findUser(Integer userId) {
+  public User findUser(Integer facultyNumber) {
     for (User user : userList) {
-      if (user.getId().equals(userId)) {
+      if (user.getFacultyNumber().equals(facultyNumber)) {
```

Comment:
We can be as certain as we make ourselves to be. The value is the uniqueness of it and the fact that students are identified by unique faculty numbers thus giving us the ability to identify users by an ID that is unique to the app and different than the id given by datastore.